LethalAPI.Lobby/LethalAPI.Lobby.csproj

```diff
@@ -1,12 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netstandard2.1</TargetFramework>
-        <AssemblyName>LethalCompanyTemplate</AssemblyName>
-        <Description>A template for Lethal Company</Description>
+        <TargetFramework>net48</TargetFramework>
+        <Description>Lobby-related functionality for LethalAPI</Description>
         <Version>1.0.0</Version>
```

Comment:
Might be an idea to specify here a 0.x.x version as per SemVer Pre-release/Development builds